src/test/java/com/google/api/codegen/testdata/go_main_library.baseline

```diff
@@ -148,14 +148,14 @@ type Client struct {
 // resources and Book resources in the library. It defines the following
 // resource model:
 //
-// - The API has a collection of [Shelf][google.example.library.v1.Shelf]
-//   resources, named ``bookShelves/*``
+//   The API has a collection of [Shelf][google.example.library.v1.Shelf]
+//   resources, named bookShelves/*
 //
-// - Each Shelf has a collection of [Book][google.example.library.v1.Book]
-//   resources, named `bookShelves/*/books/*`
+//   Each Shelf has a collection of [Book][google.example.library.v1.Book]
+//   resources, named bookShelves/*/books/*
 //
-// Check out [cloud docs!](/library/example/link).
-// This is [not a cloud link](http://www.google.com).
+// Check out cloud docs!.
```

Comment:
I think the Go docs would be poorer for the lack of these URLs. What if we render them as

*text* (at *URL*)